pkg/node/node.go

```diff
@@ -292,6 +293,20 @@ func NewNode(
 		version.LogUpdateResponse,
 	)
 
+	if !bacalhauConfig.DisableAnalytics {
+		installationID, err := fsr.ReadInstallationID()
+		if err != nil {
+			log.Trace().Err(err).Msg("failed to read installationID")
+		}
+		if err := analytics.SetupAnalyticsProvider(ctx,
+			analytics.WithNodeNodeID(config.NodeID),
+			analytics.WithNodeType(config.IsRequesterNode, config.IsComputeNode),
+			analytics.WithInstallationID(installationID),
+			analytics.WithVersion(version.Get()),
+		); err != nil {
+			log.Trace().Err(err).Msg("failed to setup analytics provider")
+		}
+	}
```

Comment:
will this cause issues with devstack? previously we only enabled analytics through `bacalhau serve`. Doing in node is fine, but we need to make sure all test paths have analytics disabled

Reply:
I'll just move the logic to serve

Reply:
still, please make sure tests don't trigger telemetry. we still have tests that call `serve`